.github/actions/lint-action/action.yml

```diff
@@ -34,6 +34,10 @@ runs:
       shell: bash
       run: pnpm install
 
+    - name: Build
```

Comment:
If the lint step requires the build artifacts check out our nx.json, specifically the `test.dependsOn` line. :)

Reply:
please elaborate, i don't get it

Reply:
If linting fails due to the build step it would make a lot of sense to add a config like test has but for lint.
![image](https://user-images.githubusercontent.com/61207713/229739297-3217e322-770e-4e72-ab6f-4c721ef7a6b8.png)

Reply:
indeed it does, moved there